readme.md

```diff
@@ -80,7 +80,9 @@ Generate rest layer? | Description | Y | n |
 What database do you want to use? | Use arrow keys to choose your Database. | -- | -- |
 Specify your entities folder path? (optional) | If you have [Gotu](https://github.com/herbsjs/gotu) entities, specify the folder path for CLI generate the project around that | empty | ./example/my-custom-entities/ |
  
+## Update
 
+If you need add some new entity after had generate the project, just add it into entity folder and type into terminal `herbs update`, so the herbs-cli will generate all the usecases, repositories and others necessary layers for this entity.
```

Comment:
Hello @italojs, I believe this piece of documentation alone does not fully explain the functionality.

Could you add more details, including troubleshooting?

Reply:
As I said into PR description, I'm open for sugestions, I can't think into a better documentation, but i will try do something better